VideoAPI/VideoApi.DAL/Commands/AddRoomParticipantCommand.cs

```diff
@@ -0,0 +1,47 @@
+using Microsoft.EntityFrameworkCore;
+using VideoApi.DAL.Commands.Core;
+using VideoApi.DAL.Exceptions;
+using VideoApi.Domain;
+using Task = System.Threading.Tasks.Task;
+
+namespace VideoApi.DAL.Commands
+{
+    public class AddRoomParticipantCommand : ICommand
+    {
+        public long RoomId { get; set; }
```

Comment:
I dont think we need setters on the properties of the `ICommand`

Reply:
done

Reply:
done